### ISSUE TITLE
[BUG] Slow Bolt content loading.

### ISSUE THREAD

**Issue by slummer87** [bolt/bolt#5235]
Sometimes Bolt loads very slow (11s). Don't have any idea why. Do you have this type of problems with your projects?

Details
-------

 - Relevant Bolt Version: master
 - Install type: Composer install
 - PHP version: 5.6
 - Used webserver: Apache
 - For UX/UI issues: Safari (latest)

<img width="1116" alt="screen shot 2016-04-21 at 19 57 16" src="https://cloud.githubusercontent.com/assets/4402959/14719454/92c1c982-07fc-11e6-9c3e-8e8cbd50db3e.png">

**Comment by GawainLynch** [comment]
That is so not the image that I got in the email :laughing: 

That session delay is a good hint… is the cache directory local and not symlinked?

**Comment by slummer87** [comment]
Yes, I know, sorry for the image. I don't change anything in cache. Default settings and directories.

**Comment by SahAssar** [comment]
Sorry, for the very, very late reply... Are you still experiencing these issues? If so, could you check in your apache/php logs for anything that seems off? I'd assume that it is related to the sessions which we now store in the cache folder, so checking permissions on that wouldn't hurt either.

**Comment by SahAssar** [comment]
Closing due to no response... Feel free to reopen if the issue is present in latest stable and you have more info!

**Comment by svivian** [comment]
I have the same issue sporadically. I ran xdebug profiling on a basic page load (home page) and the consistent issue is `Symfony\Component\Debug\DebugClassLoader->loadClass` taking up a significant portion of the load time. This is not due to the debug bar as I turned it off (set `debug: false` in the config) before running the profiling. I tried `composer install -o` to optimize the autoloading but still get the same results.

Should Bolt be using the class `DebugClassLoader` when not in debug mode? I thought most classes would be loaded via Composer.

Here's a screenshot of the profiling results from webgrind if it helps:

![bolt-slowdown](https://user-images.githubusercontent.com/207887/28547122-57a1728e-70c5-11e7-8c3b-3498999c041e.png)

I'm using the latest verison of Bolt (3.2.16) with PHP 5.5
